packages/web/src/app/Footer.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { DialogRef } from '../shared/components/Dialog';
+import { Icon } from '../shared/components/Icon';
+import LanguageDialog from './LanguageDialog';
+import interfaceLanguages from './languages.json';
+import useLocalStorageState from '../shared/hooks/useLocalStorageState';
+
+export default function Footer() {
+  const [isFirstLoad, setIsFirstLoad] = useLocalStorageState<boolean>(
+    'needs-language-dialog',
```

Comment:
I ended up using a new local storage key. See this: https://github.com/arrocke/gloss-translation/pull/107#discussion_r1231607331